Assets/MixedRealityToolkit-SDK/Features/Audio/Influencers/AudioInfluencerController.cs

```diff
@@ -0,0 +1,297 @@
+            }
+        }
+
+        /// <summary>
+        /// Finds the IAudioInfluencer objects that are to be applied to the audio source.
+        /// </summary>
+        /// <returns>Collection of IAudioInfluencers between the user and the game object.</returns>
+        private List<IAudioInfluencer> GetInfluencers()
+        {
+            List<IAudioInfluencer> influencers = new List<IAudioInfluencer>();
+            Transform cameraTransform = CameraCache.Main.transform;
+
+            // Influencers take effect only when between the emitter and the user.
+            // Perform a raycast from the user toward the object.
+            Vector3 direction = (gameObject.transform.position - cameraTransform.position).normalized;
+            float distance = Vector3.Distance(cameraTransform.position, gameObject.transform.position);
+
+            int count = Physics.RaycastNonAlloc(cameraTransform.position,
+                                                direction,
+                                                hits,
+                                                distance,
+                                                Physics.DefaultRaycastLayers,
+                                                QueryTriggerInteraction.Ignore);
+            
+            for (int i = 0; i < count; i++)
+            {
+                IAudioInfluencer influencer = hits[i].collider.gameObject.GetComponentInParent<IAudioInfluencer>();
+                if (influencer != null)
+                {
+                    influencers.Add(influencer);
+                }
+            }
+
+            return influencers;
+        }
+
+        /// <summary>
+        /// Gets the native cutoff frequency of the attached low pass filter.
+        /// </summary>
+        /// <returns>
+        /// The native cutoff frequency, or a neutral frequency if there is no low pass filter attached.
+        /// </returns>
+        public float GetNativeLowPassCutoffFrequency()
+        {
+            return nativeLowPassCutoffFrequency;
+        }
+
+        /// <summary>
+        /// Sets the cached native cutoff frequency of the attached low pass filter.
+        /// </summary>
+        /// <param name="frequency">The new low pass filter cutoff frequency.</param>
+        /// <remarks>This method may be called by an attached effect to change the native behavior
+        /// of the low pass filter for scenarios such as simulating a dynamic sound source quality change.</remarks>
+        public void SetNativeLowPassCutoffFrequency(float frequency)
+        {
+            nativeLowPassCutoffFrequency = frequency;
+        }
+
+        /// <summary>
+        /// Gets the native cutoff frequency of the attached high pass filter.
+        /// </summary>
+        /// <returns>
+        /// The native cutoff frequency, or a neutral frequency if there is no high pass filter attached.
+        /// </returns>
+        public float GetNativeHighPassCutoffFrequency()
```

Comment:
Also can be a property

Reply:
turned into a property... do not know why github did not autoresolve

Reply:
Just pulled your last commit, seems to still be a method

Reply:
somehow missed deleting htese

Reply:
No worries! Just wanted to double check :)